Batteries/Tactic/Lint/Basic.lean

```diff
@@ -85,28 +86,29 @@ initialize stdLinterExt :
   }
 
 /--
-Defines the `@[std_linter]` attribute for adding a linter to the default set.
-The form `@[std_linter disabled]` will not add the linter to the default set,
+Defines the `@[batteries_linter]` attribute for adding a linter to the default set.
+The form `@[batteries_linter disabled]` will not add the linter to the default set,
 but it will be shown by `#list_linters` and can be selected by the `#lint` command.
 
 Linters are named using their declaration names, without the namespace. These must be distinct.
 -/
-syntax (name := std_linter) "std_linter" &" disabled"? : attr
+syntax (name := batteries_linter) "batteries_linter" &" disabled"? : attr
```

Comment:
```suggestion
syntax (name := env_linter) "env_linter" &" disabled"? : attr
```
The name `std_linter` was never very good to begin with. We could do this separately but it would be nice if people didn't have to rename all the downstream linters twice...